response_operations_ui/forms.py

```diff
@@ -45,6 +45,7 @@ class EditContactDetailsForm(FlaskForm):
     last_name = StringField('last_name')
     email = StringField('emailAddress')
     telephone = StringField('telephone')
+    hidden_email = HiddenField('hidden_email')
```

Comment:
If this is just the user's present email address it doesn't look like you need this field. Could you not just use the email address you get back when making the 'get_contact_details(respondent_id)' call on line 38 in contact_details_controllers.py?